### ISSUE TITLE
Accessibility Audit: Improper announcement of page title when user navigates to Log In page (Windows/Firefox)

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#942]
**ENVIRONMENT**

Windows 10 Enterprise | Firefox Version 95.0 (64-bit) | JAWS
Windows 10 Enterprise | Firefox Version 95.0 (64-bit) | NVDA
Windows 10 Enterprise | Chrome Version 96.0.4664.45 (Official Build) (64-bit) | NVDA

**ISSUE**

Turn on the screen reader, and go to the [Log In](https://detroit-public-dev.netlify.app/sign-in) page, and listen to the announcement. The Screen reader doesn't announce the page title in a proper manner when user navigates to Sign In page. Voiceover, Chromevox, JAWS & NVDA announce Detroit Home Connect, which is the correct page title. However, JAWS/ NVDA with Firefox browser announce some gibberish data. The same issue is observed in the Account Settings page. Voiceover announces Detroit Home Connect. NVDA with chrome browser and JAWS/ NVDA with Firefox browser doesn't announce any page title.
